### ISSUE TITLE
overwrite or delete specific JVM opts?

### ISSUE THREAD

**Issue by githorse** [technomancy/leiningen#2739]
I'm getting the `-Xverify:none is deprecated in JDK 13` error from #2611. I understand from that thread that the default JVM opts are not critical and I can just overwrite them, but I'd prefer not to; I'd rather let leiningen figure those out for me and only tweak what I need. So, is there a way to only overwrite specific options, without overwriting the whole `JVM_OPTS` variable?

(I guess this may be covered by #1230 but it's not clear to me that's a general solution; seems to be addressing the `Tiered*` options specifically.)

**Comment by technomancy** [comment]
Previously that flag was required to improve startup time. However, across more recent tests I've done, it doesn't seem to have any effect on startup time any more.

I'll remove it so this is no longer an issue.

**Comment by githorse** [comment]
@technomancy - Ok, to clarify, you're saying that going forward there are will be no default JVM opts at all?

**Comment by technomancy** [comment]
No, the tiered compilation settings remain. But if you want to turn off
tiered compilation then you probably want to replace all the settings.

We have a lot of flexibility in how we handle options once Leiningen has
launched, but because of the bootstrapping problem, we're stuck with
using bash to launch Leiningen itself. Working in a language that
literally does not have the concept of a data structure sucks, and this
is one of the downsides.
